.github/pull_request_template.md

```diff
@@ -1,24 +1,28 @@
-### Name of the story
+# A nice friendly title for what this PR does
 
-[Airtable ticket](https://airtable.com/####)
+Relates to issue #(an issue)/bug found due to #(previous pull request).
 
 ![]('gif_link_here')
 
 ### Acceptance Criteria
 
-- [ ] ...
+- [ ] _The first thing this PR must achieve
+- [ ] _The second thing this PR must achieve
+- [ ] _etc_
 
 ### Testing information
 
-<insert notes for testers that might be helpful>
+_notes that might be helpful for testing_
 
 ### Pre-flight Checklist
 
-- [ ] Tests
+- [ ] Testing aligns with [testing strategy][testing strategy]
 - [ ] DUXD review approval
 - [ ] Accessibility Reviewed
 - [ ] Product owner demo
 
 ### Pre-merge Checklist
 
 - [ ] Branch test approved
+
+[testing strategy]: https://github.com/MHRA/products/blob/master/docs/principles/testing.md "MHRA/products testing strategy"
```

Comment:
Is there a benefit to having this link here, rather than inline where it's mentioned above?

Reply:
It's a habit of mine in markdown for three reasons:

1. It makes the paragraphs more readable not to have to read past potentially lengthy URLs and their titletext
2. It allows for reusing URLs
3. It puts the part of the file for updating them into one spot, like declaring hardcoded `const` at the top of a `.js`